git_apple_llvm/am/am_graph.py

```diff
@@ -11,6 +13,23 @@
     pass
 
 
+def is_blocked(config: AMTargetBranchConfig, remote: str = 'origin'):
+    if not config.common_ancestor:
+        return False
+    commit_log_output = git_output(
+        'log',
+        '--first-parent',
+        '--pretty=format:%H %cd', '--no-patch',
```

Comment:
If you do format '%H', you won't need to do the second split to find the hash, as you'll be listing hashes only without the dates.